draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -55,47 +55,47 @@ informative:
 
 --- abstract
 
-Existing HTTP authentication mechanisms are probeable in the sense that it is
+Existing HTTP authentication schemes are probeable in the sense that it is
```

Comment:
Is your abstract really just a repetition of the first paragraph of the introduction?

Maybe you can instead say what the document does (the last sentence, basically).

Reply:
Yeah that was somewhat lazy. I plan on fixing that once the draft gets a bit more stable

Reply:
Filed #2577 to keep track of this